internal/service/keyspaces/table.go

```diff
@@ -204,7 +204,7 @@ func ResourceTable() *schema.Resource {
 										ValidateFunc: validation.All(
 											validation.StringLenBetween(1, 48),
 											validation.StringMatch(
-												regexp.MustCompile(`^[a-z0-9][a-z0-9_]{1,47}$`),
+												regexp.MustCompile(`^[a-z0-9][a-z0-9_]{0,47}$`),
```

Comment:
```suggestion
												regexp.MustCompile(`[a-zA-Z0-9][a-zA-Z0-9_]{0,47}$`),
```

Looks like the original was taken directly from the [API docs](https://docs.aws.amazon.com/keyspaces/latest/APIReference/API_TableSummary.html). I think the above might be more accurate; insist on one character then 0 or more up to a maximum complete length of 48?

Reply:
It looks like i had hit the commit suggestions before you edited your suggestion comment, ive push an update for the above edited suggestion.